src/ElementData.h

```diff
@@ -9,27 +9,70 @@
 #define DRAFTER_ELEMENTDATA_H
 
 #include "RefractSourceMap.h"
+#include <deque>
 
 namespace drafter
 {
+    template <typename ElementT, typename = std::enable_if<std::is_base_of<refract::IElement, ElementT>::value> >
+    constexpr bool is_primitive = false;
```

Comment:
not sure about introduce new naming conventions which does not conform to current conventions.
I personally prefer std c++ naming conventions, but this is team project

Reply:
@kylef @pksunkara what do you think?